src/sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution.ts

```diff
@@ -42,6 +42,12 @@ configurationRegistry.registerConfiguration({
 		'datasource.connectionGroups': {
 			'description': localize('datasource.connectionGroups', "data source groups"),
 			'type': 'array'
+		},
+		'datasource.connectionSort': {
+			'type': 'string',
+			'enum': ['by time added', 'by title alphabetically'],
```

Comment:
The convention we follow is that the enum values are camel cased with no spaces (e.g. `byTimeAdded` and `byTitleAlphabetically`. You can do a search for `'enum':` to see other examples of this in the code base

Reply:
updated